### DISCUSSION TITLE
How to add auto invitation at organization profile?

### DISCUSSION THREAD

**Discussion by noahstoner** [Finalgoal231/finalGoal#73] [New Ideas for orgs management] [answered]
For free join, we should add auto invitation.
Let's discuss about this problem.
There are steps to join organization:
Step 1: Create auto issue per user.
Step 2: Close issue and send invite link to notification inbox.
Step 3: User accept invite link.

**Comment by creative2113** [comment]
Oh, I see.
I have done.
it is very help for me.
thanks.

**Comment by creative2113** [comment]
Hello, @shinevue 

how do you think about that we have to send discord id when we get the invitation from organization?
please give me the best proposal.
thanks.

**Comment by creative2113** [comment]
I can perfectly complete that.
thanks.

**Reply by popstar7** [reply]
If you can, please share your code here to invite at our organization.

**Reply by creative2113** [reply]
do you really want the code?

```js

const invite = {
  click: "link",
  title: "I want to join this server.",
  discord_id: "XXX",
  description: "...",
}
```
please check this.

**Reply by popstar7** [reply]
We should add auto-invite code at profile readme file.
But this code's type isn't match with readme.
Please search this link:
https://github.com/Magic-Academy/.github/blob/main/profile/README.md?plain=1

**Reply by creative2115** [reply]
I am interesting.
you are discussing about great problem.
I want to hear about that.

**Reply by creative2113** [reply]
> We should add auto-invite code at profile readme file. But this code's type isn't match with readme. Please search this link: https://github.com/Magic-Academy/.github/blob/main/profile/README.md?plain=1

I see.

**Reply by creative2113** [reply, marked as answer]
I solved this problem.

- at first, we have to create issue template
- second, copy issue template link
- next, add link to github organization profile

please check and go ahead.

**Comment by creative2115** [comment]
👋Hello, @shinevue 
I want to join your server.
please send me invitation for me.

**Reply by creative2115** [reply]
👋Hello, @shinevue 
can you manually send me invitation link, anyway?

**Comment by Daniel-Kennison** [comment]
First create a link of invitation.
You can find link creation rule.
